pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -37,8 +37,9 @@ type preScoreState struct {
 	Constraints []topologySpreadConstraint
 	// IgnoredNodes is a set of node names which miss some Constraints[*].topologyKey.
 	IgnoredNodes sets.Set[string]
-	// TopologyPairToPodCounts is keyed with topologyPair, and valued with the number of matching pods.
-	TopologyPairToPodCounts map[topologyPair]*int64
+	// TopologyValueToPodCounts is a slice indexed by constraint index.
```

Comment:
Fix comment below, which still refers to 'TopologyPairToPodCounts'

Reply:
I think the comment is correct